.github/workflows/linux-eic-shell.yml

```diff
@@ -116,9 +118,14 @@ jobs:
           !.git/
           !.ccache/
         if-no-files-found: error
+    # Only compress and upload build directory if we are going to use it later
     - name: Compress build directory
+      if: ${{ matrix.CC }} == ${{ env.clang-tidy-iwyu-CC }}
+       && ${{ matrix.CMAKE_BUILD_TYPE }} == ${{ env.clang-tidy-iwyu-CMAKE_BUILD_TYPE }}
```

Comment:
Huh, it seems GitHub actions accepts multi-lines without throwing a syntax error, but then it just ignores the second line... Both clang entries in the matrix got their build directory uploaded...

Reply:
And it uploaded the gcc build directories too. Something's wrong.

Reply:
Fixed in ae79bc8.